### PR TITLE
Mobile

### PR DESCRIPTION
**Pull request by DmitriiTsy**

### CODE REVIEW

**Comment by DmitriiTsy** [comment]
I fixed some major issues such as naming, tags (I changed it from  the divs to regular - "h, header, span etc") Should be helpful for semantic optimisation

There are some minor issues with spaces that I will fix with your new feedback

File: main.css [hunk 1]
```diff
@@ -0,0 +1,771 @@
...
+.header {
+    display: flex;
+    height: 220px;
+    border-bottom: 0.5px solid grey;
+    justify-content: center;
+    align-items: center;
+    gap: 40px
+}
+
+.header__glass{
```
Comment (ghost) [inline main.css]: missing space

File: main.css [hunk 2]
```diff
@@ -0,0 +1,771 @@
...
+    left: 40px;
+}
+
+.header__glass-source{
+    width: 35px;
+    height: 35px;
+    position: absolute;
+}
+
+.header__small-container{
```
Comment (ghost) [inline main.css]: what is an element and what is a modificator here? small sounds like a modificator? doesn't it?

File: main.css [hunk 3]
```diff
@@ -0,0 +1,771 @@
...
+    display:flex;
```
Comment (ghost) [inline main.css]: space missed

File: main.css [hunk 4]
```diff
@@ -0,0 +1,771 @@
...
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+    justify-content: center;
+    flex-direction: column;
+    gap: 80px;
+    padding: 100px 0px 100px 0px;
+}
+
+.let-ride h1 {
```
Comment (ghost) [inline main.css]: tag selector, avoid this

File: main.css [hunk 5]
```diff
@@ -0,0 +1,771 @@
...
+    font-size: 100px;
+    font-weight: 550;
+}
+
+.let-ride span {
```
Comment (ghost) [inline main.css]: another tag selector

File: main.css [hunk 6]
```diff
@@ -0,0 +1,771 @@
...
+.drive-with {
+    display: flex;
+    text-align: center;
+    flex-direction: column;
+    gap: 10px;
+    background-color: #F4F4F9;
+    padding: 200px 0px 0px 0px;
+}
+
+.drive-with span {
```
Comment (ghost) [inline main.css]: you know what to do with that :D

File: main.css [hunk 7]
```diff
@@ -0,0 +1,771 @@
...
+    font-size: 40px;
+    font-weight: 600;
+    width: 100%;
+} 
+
+.drive-with p{
```
Comment (ghost) [inline main.css]: you know what to do with that :D

File: main.css [hunk 8]
```diff
@@ -0,0 +1,771 @@
...
+    display: flex;
+    flex-direction: row;
+}
+
+.drive-with__text-section{
+    display:flex;
+    flex-direction: column;
+}
+
+.drive-with__text-section p {
```
Comment (ghost) [inline main.css]: you know what to do with that :D

File: main.css [hunk 9]
```diff
@@ -0,0 +1,771 @@
...
+    font-size:40px;
+    font-weight: 300;
+}
+
+.drive-with__text-section strong {
```
Comment (ghost) [inline main.css]: you know what to do with that :D

File: main.css [hunk 10]
```diff
@@ -0,0 +1,771 @@
...
+    justify-content: center
+}
+
+.drive-with__how-driver-pay-container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.drive-with__how-driver-pay-container span{
```
Comment (ghost) [inline main.css]: you know what to do with that :D

File: main.css [hunk 11]
```diff
@@ -0,0 +1,771 @@
...
+.drive-with__feedback {
+    display:flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    gap: 40px;
+    justify-content: center;
+    align-items: center;
+}
+
+.drive-with__feedback p {
```
Comment (ghost) [inline main.css]: you know what to do with that :D

File: main.css [hunk 12]
```diff
@@ -0,0 +1,771 @@
...
+    font-size: 40px;
+    font-weight: 400;
+    line-height: 1.5;
+}
+
+.drive-with__feedback span {
```
Comment (ghost) [inline main.css]: you know what to do with that :D

File: main.css [hunk 13]
```diff
@@ -0,0 +1,771 @@
...
+    padding-left: 30px;
+}
+
+.ride__point{
+    display:flex;
+    flex-direction: row;
+    gap: 30px;
+}
+
+.ride span {
```
Comment (ghost) [inline main.css]: you know what to do with that :D

File: main.css [hunk 14]
```diff
@@ -0,0 +1,771 @@
...
+    font-size: 40px;
+    font-weight: 600;
+    width: 100%;
+}
+
+.ride h2 {
```
Comment (ghost) [inline main.css]: you know what to do with that :D

File: main.css [hunk 15]
```diff
@@ -0,0 +1,771 @@
...
+    font-size: 70px;
+    font-weight: 600;
+    width: 100%;
+}
+
+.ride p {
```
Comment (ghost) [inline main.css]: you know what to do with that :D

File: main.css [hunk 16]
```diff
@@ -0,0 +1,771 @@
...
+    font-size: 30px;
+    font-weight: 300;
+    width: 100%;
+}
+
+.ride__list strong {
```
Comment (ghost) [inline main.css]: you know what to do with that :D

File: main.css [hunk 17]
```diff
@@ -0,0 +1,771 @@
...
+    flex-direction: row;
+    gap: 140px
+}
+
+.ride__slider-arrows{
+    width: 100px;
+    height: 100px;
+}
+
+.ride__cards p{
```
Comment (ghost) [inline main.css]: the same comment for the rest tag selectors

File: main.css [hunk 18]
```diff
@@ -0,0 +1,771 @@
...
+    justify-content: center;
+}
+
+/* Estimation pink section */
+
+.estimation{
+    display:flex;
+    flex-direction: column;
+    text-align: left;
+    background-color:#DCDCFF;
```
Comment (ghost) [inline main.css]: space is missing between rule name and rule value, make sure you fix it everywhere

File: main.css [hunk 19]
```diff
@@ -0,0 +1,771 @@
...
+    border: 1px solid black;
+    font-size: 45px;
+    padding: 40px 10px 0px 0px;
+    display: flex;
+    position: relative;
+    text-align: center;
+}
+
+.estimation__textareas{
+    display:flex;
```
Comment (ghost) [inline main.css]: you can also fix this by installing Stylelint plugin to VS Code

File: main.html [hunk 20]
```diff
@@ -1,9 +1,367 @@
...
-      <TITLE>My first HTML document</TITLE>
-   </HEAD>
-   <BODY>
-      <P>Hello world!
-   </BODY>
-</HTML>
+<!DOCTYPE html>
+<html>
+<head>
+<title>Page Title</title>
```
Comment (ghost) [inline main.html:5]: let's add normalize.css here as we discussed

**Comment by ghost** [comment]
> I fixed some major issues such as naming, tags (I changed it from the divs to regular - "h, header, span etc") Should be helpful for semantic optimisation
> 
> There are some minor issues with spaces that I will fix with your new feedback

let's fix all minor issues before sending a PR, it's very important for reviewers to be able to review 'prepared' PRs and don't spend extra time on this. Mind adding a screenshot? for the long websites, you can use https://chrome.google.com/webstore/detail/gofullpage-full-page-scre/fdpohaocaechififmbbbbbknoalclacl

**Comment by ghost** [comment]
I noticed that you used `first` word for repo naming, let's avoid numbers for naming,`lyft-main-page` or `lyft-main` would be better (do not change anything, just a comment for the future)

**Comment by DmitriiTsy** [comment]
PDF Version  [Lyft__mobile.pdf](https://github.com/DmitriiTsy/Lyft_first/files/8882382/Lyft__mobile.pdf)

Mobile Version Lyft__mobile

Hi!, sure. This is the full screenshot of a new version with fixes:

1. I deleted all tags (p, span, etc)  in CSS and added classes;
2. Fixed all white spaces in the CSS;
3. Fixed more issues regarding page (spaces between elements, font sizes, etc)
4. I added the newest CSS normalize at the start of the page.  

@anatoliy8493

**Comment by DmitriiTsy** [comment]
I also have one specific question :) Is it important to use <footer>, <header> and etc. instead of "div" everywhere on the page? If yes, I will add them. 

The reason I asking is that I only added semantic text tags (p, span, h1, h2 and etc) but not all semantic tags

**Comment by DmitriiTsy** [comment]
Hi, @anatoliy8493 Added a new commit ("Normalize and semantic tags"): 

- Added normalize.css instead of reset.css that was before;
- Added more semantic tags;

* Attached w3 HTML validator analysis (errors in HTML tags and etc.) 

Screen Shot 2022-06-13 at 15 07 08

File: main.css [hunk 21]
```diff
@@ -0,0 +1,789 @@
+html,
+body {
+    font-family: sans-serif;
+    margin: 0px;
```
Comment (ghost) [inline main.css:4]: do we still need this after normilize.css was included?

File: main.css [hunk 22]
```diff
@@ -0,0 +1,789 @@
...
+    gap: 100px;
+}
+
+.drive-with__how-driver-pay-container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.drive-with__bold {
```
Comment (ghost) [inline main.css:191]: mixed up element and modificator, read https://en.bem.info/methodology/naming-convention/

File: main.html [hunk 23]
```diff
@@ -1,9 +1,356 @@
...
-      <P>Hello world!
-   </BODY>
-</HTML>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="UTF-8">
+<title>Page Title</title>
+<link rel="stylesheet" href="main.css">
+<link rel="stylesheet" href="normalize.css">
```
Comment (ghost) [inline main.html:7]: normalize should be the 1st

File: normalize.css [hunk 24]
```diff
@@ -0,0 +1,349 @@
+/*! normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css */
+
+/* Document
```
Comment (ghost) [inline normalize.css:3]: can we load it from CDN and avoid storing at the repo?

**Review by ghost — APPROVED** [review]
LGTM once we fix the last comments

**Comment by DmitriiTsy** [comment]
Hi

I added     <meta content='width=device-width, initial-scale=1.0, maximum-scale=1.0,
user-scalable=0' name='viewport' />

To my HTML in order to create responsive content on the page, depending on the device width. But after that, all of my previous work with layouts crashed.

Is it a best practice to use these meta tags all the time?  @anatoliy8493

**Comment by DmitriiTsy** [comment]
Screen Shot 2022-06-15 at 12 24 58

**Comment by DmitriiTsy** [comment]
screencapture-192-168-1-9-56526-main-html-2022-06-16-15_26_27


Apple Ipad Mini

**Comment by DmitriiTsy** [comment]
screencapture-192-168-1-9-56526-main-html-2022-06-16-15_28_32

Tablet (wide) - Nest Hub Max

**Comment by DmitriiTsy** [comment]
screencapture-192-168-1-9-56526-main-html-2022-06-16-15_29_52

MacBook  13" (DeskTop)

**Comment by DmitriiTsy** [comment]
Hi! 

I added 2 tablet versions and 1 desktop.  (Didn't create a cards section - because it depends on the flow and js as I understand right) 

@anatoliy8493 

What do you think?

**Pull request merged**